generators/app/index.js

```diff
@@ -108,6 +109,40 @@ module.exports = yeoman.generators.Base.extend({
       done();
     },
 
+    graphicsmagickExists: function () {
+      var done = this.async();
+
+      var self = this;
+
+      if (process.platform == 'darwin') {
```

Comment:
Nope. Doesn't work, `done()` doesn't stop it and return stops the whole process.